src/net/sourceforge/kolmafia/request/CurseRequest.java

```diff
@@ -23,7 +23,6 @@ public class CurseRequest extends GenericRequest {
 
   private final AdventureResult itemUsed;
 
-  private static final AdventureResult SMORE_GUN = ItemPool.get(ItemPool.SMORE_GUN, 1);
```

Comment:
This is part of cleaning up CurseRequest and nothing to do with the training manual, correct?  Only a negative answer is significant.

Reply:
Correct.